src/lib/Helpers.coffee

```diff
@@ -100,12 +116,16 @@ exports.WirePattern = (component, config, proc) ->
   config.forwardGroups = false unless 'forwardGroups' of config
```

Comment:
I wonder if we should swap the default now to `true`, just like `forwardBrackets`